### PR TITLE
Update of installing-openvino-pip.md

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Adding more detailed installation procedures while using PyPi distribution of OpenVINO Toolkit. Removal of duplicate content, regarding installation of OpenVINO Development Tools which is elaborated in a separate article of the documentation.

### Details:
 - *item1*
 - *...*

### Tickets:
 - *ticket-id*

### CODE REVIEW

**Review by xu-yuan1 — CHANGES_REQUESTED** [review]
Hi @sgolebiewski-intel , this file is also updated in this PR: https://github.com/openvinotoolkit/openvino/pull/10980/files#diff-0795b798624c67963281774fa416021996fde4015d622fe88932f621dfef88b5. To avoid conflicts, could you please add the "System Requirements" and "Troubleshooting" sections only and copy the rest of it from #10980 as it's already reviewed?

File: docs/install_guides/installing-openvino-pip.md [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
-You can install Intel® Distribution of OpenVINO™ Toolkit Runtime Package through the PyPI repository. 
+You can install both OpenVINO™ Runtime and OpenVINO Development Tools through the PyPI repository. This page provides the main steps for installing OpenVINO Runtime.
 
-## System Requirements
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
+## Installing OpenVINO Runtime
 
-#### Below you will find all supported operating systems and required Python\* versions. 
+### System requirements
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:8]:
```suggestion
### System Requirements
```

**Comment by sgolebiewski-intel** [comment]
> Hi @sgolebiewski-intel , this file is also updated in this PR: https://github.com/openvinotoolkit/openvino/pull/10980/files#diff-0795b798624c67963281774fa416021996fde4015d622fe88932f621dfef88b5. To avoid conflicts, could you please add the "System Requirements" and "Troubleshooting" sections only and copy the rest of it from #10980 as it's already reviewed?

I've done as you suggested. Thank you for your help.

File: docs/install_guides/installing-openvino-pip.md [hunk 2]
```diff
@@ -22,113 +25,115 @@ You can install Intel® Distribution of OpenVINO™ Toolkit Runtime Package thro
...
+python -c "from openvino.runtime import Core"
 ```
 
-## Troubleshooting
+If installation was successful, you will not see any error messages (no console output).
+
+### Troubleshooting
 
-### You may be prompted with the following errors:
+You may be prompted with the following errors:
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:95]:
```suggestion
Here you may find solutions to some common issues.
```

File: docs/install_guides/installing-openvino-pip.md [hunk 3]
```diff
@@ -22,113 +25,115 @@ You can install Intel® Distribution of OpenVINO™ Toolkit Runtime Package thro
...
 
 @sphinxdirective
 
-.. tabs::
+.. tab:: Linux and macOS
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:100]:
```suggestion

   Error message:

```

File: docs/install_guides/installing-openvino-pip.md [hunk 4]
```diff
@@ -1,10 +1,13 @@
...
+You can install both OpenVINO™ Runtime and OpenVINO Development Tools through the PyPI repository. This page provides the main steps for installing OpenVINO Runtime.
 
-## System Requirements
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
+## Installing OpenVINO Runtime
 
-#### Below you will find all supported operating systems and required Python\* versions. 
+### System requirements
+
+Below you will find all supported operating systems and required Python\* versions.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:10]:
```suggestion
Below you can find all the supported operating systems and the required Python versions. 
```

File: docs/install_guides/installing-openvino-pip.md [hunk 5]
```diff
@@ -1,10 +1,13 @@
...
-## System Requirements
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
+## Installing OpenVINO Runtime
 
-#### Below you will find all supported operating systems and required Python\* versions. 
+### System requirements
+
+Below you will find all supported operating systems and required Python\* versions. 
 
 |Operating Systems (64 bit)                        | [Python\* Version](https://www.python.org/downloads/) (64 bit)                 |
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:12]:
```suggestion
|Operating System (64-bit)                        | [Python Version](https://www.python.org/downloads/) (64-bit)                 |
```

File: docs/install_guides/installing-openvino-pip.md [hunk 6]
```diff
@@ -22,113 +25,115 @@ You can install Intel® Distribution of OpenVINO™ Toolkit Runtime Package thro
...
 
 @sphinxdirective
 
-.. tabs::
+.. tab:: Linux and macOS
 
-    .. tab:: WINDOWS\*
+   *ImportError: libpython3.7m.so.1.0: cannot open shared object file: No such file or directory*
+   
+   To resolve missing external dependency on Ubuntu* 18.04, execute the following command:
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:103]:
```suggestion
   A possible cause is missing the external dependency on Ubuntu 18.04. To resolve this, run the following command:
```

File: docs/install_guides/installing-openvino-pip.md [hunk 7]
```diff
@@ -22,113 +25,115 @@ You can install Intel® Distribution of OpenVINO™ Toolkit Runtime Package thro
...
-      To resolve missing external dependency on Ubuntu* 18.04, execute the following command:
+.. tab:: Windows
 
-      ```
-      sudo apt-get install libpython3.7
-      ```
+   *Microsoft Visual C++ 14.0 is required. Get it with "Build Tools for Visual Studio"*
 
-@endsphinxdirective
+   To resolve this issue, you need to install [Build Tools for Visual Studio* 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and repeat package installation.
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:113]:
```suggestion
   To resolve this issue, you need to install [Build Tools for Visual Studio 2019](https://visualstudio.microsoft.com/downloads/#build-tools-for-visual-studio-2019) and then retry the installation steps.
```

File: docs/install_guides/installing-openvino-pip.md [hunk 8]
```diff
@@ -1,10 +1,13 @@
-# Install Intel® Distribution of OpenVINO™ Toolkit from PyPI Repository {#openvino_docs_install_guides_installing_openvino_pip_win}
+# Install Intel® Distribution of OpenVINO™ Toolkit from PyPI Repository {#openvino_docs_install_guides_installing_openvino_pip}
 
-You can install Intel® Distribution of OpenVINO™ Toolkit Runtime Package through the PyPI repository. 
+You can install both OpenVINO™ Runtime and OpenVINO Development Tools through the PyPI repository. This page provides the main steps for installing OpenVINO Runtime.
 
-## System Requirements
+> **NOTE**: From the 2022.1 release, the OpenVINO™ Development Tools can only be installed via PyPI. See [Install OpenVINO Development Tools](installing-model-dev-tools.md) for detailed steps.
+## Installing OpenVINO Runtime
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:6]:
```suggestion

## Installing OpenVINO Runtime
```

File: docs/install_guides/installing-openvino-pip.md [hunk 9]
```diff
@@ -3,13 +3,14 @@
...
-### System requirements
+### System Requirements
 
-Below you will find all supported operating systems and required Python\* versions. 
+Below you can find all the supported operating systems and the required Python versions. 
 
-|Operating Systems (64 bit)                        | [Python\* Version](https://www.python.org/downloads/) (64 bit)                 |
+|Operating System (64-bit)                        | [Python Version](https://www.python.org/downloads/) (64-bit)                 |
 |--------------------------------------------------|--------------------------------------------------------------------------------|
 |CentOS* 7                                         | 3.6, 3.7, 3.8                                                                  |
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-pip.md:15]: I thought CentOS 7 is not supported, could you double-check? Also please help remove the asterisks (*) for brand/product names, as we are allowed to skip them in our doc site.

Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-pip.md]: Can't install it on CentOS 7. I've updated the documentation page.

File: docs/install_guides/installing-openvino-docker-windows.md [hunk 10]
```diff
@@ -5,170 +5,249 @@ This guide provides steps for creating a Docker image with Intel® Distribution
...
+  +------------------------------------+--------------------------+---------------------------------------------------------------------------------------------------+
+  | Target Operating System            | Python Version           | Host Operating System                                                                             |
+  +====================================+==========================+===================================================================================================+
+  | Windows Server Core base LTSC 2019 | 3.8                      | Windows 10 (64-bit) Pro, Enterprise, Education<br> (1607 Anniversary Update, Build 14393 or later)|
+  +------------------------------------+--------------------------+---------------------------------------------------------------------------------------------------+
+  | Windows 10, version 20H2           | 3.8                      | Windows Server 2016 or higher                                                                     |
+  +------------------------------------+--------------------------+---------------------------------------------------------------------------------------------------+
```
Comment (xu-yuan1) [inline docs/install_guides/installing-openvino-docker-windows.md:17]: Will this make it look like one target OS corresponds to one host system?

**Review by xu-yuan1 — CHANGES_REQUESTED** [review]
For the Docker installation, please do not include all the steps in a dropdown in the Installation Flow, as it does not make sense to show only two dropdown buttons without any explicit content when user opens the page. Please revert to the previous structure.

File: docs/install_guides/installing-openvino-docker-windows.md (hunk 10 above)
Comment (sgolebiewski-intel) [reply docs/install_guides/installing-openvino-docker-windows.md]: Now, that you mention it, one might actually think they're corresponding. Guess, the best way is to leave these separate as they were. I have reverted the layout. Thank you.

**Comment by sgolebiewski-intel** [comment]
Moved to #11750

**Pull request closed without merging**